app/flows/initial_request_flow.rb

```diff
@@ -0,0 +1,156 @@
+class InitialRequestFlow
+  PE_SUBJECT_CODE = "C6".freeze
+
+  attr_reader :params
+
+  def initialize(params:)
+    @params = params
+  end
+
+  def template
+    if number_of_places_page?
+      "providers/allocations/places"
+    elsif blank_search_query? || empty_search_results?
+      "initial_request"
+    elsif pick_a_provider_page?
+      "providers/allocations/pick_a_provider"
+    else
+      "initial_request"
```

Comment:
`providers/allocations/pick_a_provider`
`providers/allocations/places`

shouldn't `initial_request` be `providers/allocations/initial_request`

Reply:
these are now all explicit with full path for consistency